lib/client.js

```diff
@@ -122,9 +122,9 @@ class BPClient extends Client {
    * info for
    * @returns {Promise}
    */
-  getClientInfo(_id) {
+  getClientInfo(_id, health=false) {
     const id = _id ? _id : this.id;
-    return this.get(`/clients/${id}`);
+    return this.get(`/clients/${id}${health ? '?health=true' : ''}`);
```

Comment:
You can't pass in a second argument ` { health } ` instead of appending a string?

Reply:
Ah, actually I might be able to (https://github.com/bcoin-org/bcurl/blob/master/lib/client.js#L176) let me try it out.

Reply:
Good catch. Resolved.